connector_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqs
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/conduitio-labs/conduit-connector-sqs/common"
+	"github.com/conduitio-labs/conduit-connector-sqs/destination"
+	"github.com/conduitio-labs/conduit-connector-sqs/source"
+	testutils "github.com/conduitio-labs/conduit-connector-sqs/test"
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+)
+
+func TestFifoQueues(t *testing.T) {
```

Comment:
@alarbada this is testing we're getting the expected records from the queue, but not the actual order (FIFO). Is that intentional?

Reply:
I mean, I'm writing 10 records from 1 to 10 (just changed indexes) and then reading them 1 to 10. I might be really dumb, but first record written was 1, first record read was 1, so that's the actual order to test right?

Sadly with localstack I can't reliably reproduce the diferent sqs queue behaviours between FIFO and not FIFO. This test makes sure that the FIFO codepath is hit and it works.

Reply:
@alarbada 👍 . Maybe I was trying to be too sophisticated. This works.

Reply:
Nah no problem @raulb thx for the review